README.md

```diff
@@ -369,11 +371,11 @@ class TurkWorker
   @retry_exceptions = [NetworkError]
 
   retry_criteria_check do |exception, *args|
-    if exception.message =~ /InvalidJobId/
-      false # don't retry if we got passed a invalid job id.
-    else
-      true  # its okay for a retry attempt to continue.
+    if exception.message =~ /SpecialErrorMessageToRetry/
```

Comment:
This will not work. `false` will always be returned.

Reply:
@jzaleski nice catch! This is fixed.